iris-mpc-cpu/src/execution/hawk_main.rs

```diff
@@ -1328,7 +1328,13 @@ mod tests {
     fn receive_batch_shares(shares: Vec<GaloisRingSharedIris>) -> [IrisQueryBatchEntries; 4] {
         let mut out = [(); 4].map(|_| IrisQueryBatchEntries::default());
         for share in shares {
-            let one = preprocess_iris_message_shares(share.code, share.mask).unwrap();
+            let one = preprocess_iris_message_shares(
+                share.code.clone(),
+                share.mask.clone(),
+                share.code,
+                share.mask,
+            )
```

Comment:
I think in the test it does not matter much, but maybe we should also do this in a "correct" way here and address the TODO below already?

Reply:
Good call, yes will incorporate this